sympy/integrals/meijerint.py

```diff
@@ -1810,21 +1813,28 @@ def meijerint_definite(f, x, a, b):
         _debug('Integrand has Singularity Function terms - giving up.')
         return None
 
+    # Check if the limits are already symified or not
+    # is_eq can only be used with sympified objects
+    if not hasattr(a, '__sympy__'):
+        a = _sympify(a)
+    if not hasattr(b, '__sympy__'):
+        b = _sympify(b)
```

Comment:
If `meijerint_definite` does not sympify its arguments then the expectation must be that they should already be sympified so this should not be necessary.

Reply:
I did it because of these tests 
https://github.com/sympy/sympy/blob/62572863590b1c3363d6268f75aea6e83e385728/sympy/integrals/tests/test_meijerint.py#L77-L79
Moreover, I think it is good to have that this check, it assures that ``is_eq`` does not result in any errors.  As you have pointed out in your other comments  ``sympify`` can be very slow,  but here the objects getting sympified would just be 'int' or 'floats', I think.

Reply:
The test should be changed to use `S.Zero` instead of `0`.